orchestrator/services/report_service.py

```
@@ -0,0 +1,469 @@
+
+            logger.info(
+                "[ReportService] Created report %s for agent %s (%s): %s",
+                report_id, agent_name, report_type, title,
+            )
+
+            return {
+                "success": True,
+                "report_id": report_id,
+                "file_path": file_path,
+                "title": title,
+            }
```

Ruff (0.15.5)

[warning] Consider moving this statement to an `else` block

(TRY300)